index.html

```diff
@@ -3985,26 +4025,70 @@ <h2>
           <li>Let <var>request</var> be the <a>PaymentRequest</a> object that
```

Comment:
The algorithms seem to be switched; this is the payment details changed algorithm.

Reply:
Argh, oops... screwed up the merge/